### ISSUE TITLE
string too long  broke again?

### ISSUE THREAD

**Issue by NuggetKitten** [lorddusk/HQM#695]
I  dont expect a fix for the 1.12.2  version but it would be nice 
[latest.log](https://github.com/user-attachments/files/15525723/latest.log)

**Comment by canitzp** [comment]
yes no fix indeed, you have to fix the too long string, then this issue goes away. I think the max length is 32k character per string

**Comment by NuggetKitten** [comment]
I have gone through all the quests and the only thing that is above 30k characters is the player data itself in the config .  I am gonna see if I can get it lowered in any way
